package.json

```diff
@@ -49,11 +50,11 @@
     "babel-polyfill": "6.26.0",
     "bip39": "2.5.0",
     "bn.js": "4.11.8",
+    "bootstrap": "^4.1.3",
     "classnames": "^2.2.6",
     "clean-webpack-plugin": "^0.1.19",
     "copy-webpack-plugin": "4.6.0",
     "crypto-js": "3.1.9-1",
-    "css-loader": "^1.0.0",
```

Comment:
Why was this removed? It's still being used in the webpack config.

Reply:
Dump mistake on my part, fixed.